### DISCUSSION TITLE
最新版本的蓝灯不支持 Windows 7

### DISCUSSION THREAD

**Discussion by ycjdixon** [getlantern/lantern-forum-cn#220] [General]
蓝灯7.8.0以及升级后的7.8.1都不支持WIN7系统。旧版本也无法使用了。请问什么时候修复？

**Comment by MaxMKMax** [comment]
@ycjdixon 是的。WIN 7 在使用蓝色灯笼时遇到问题。我们也注意到了这一点。我们正在创建其他东西，以允许 Windows 7 用户使用蓝灯。请等我们。解决方案很快就会出来。

**Reply by MaxMKMax** [reply]
@ycjdixon 
您好! 想跟您分享一个好消息! 很高兴告诉您我们已经解决了Window7的问题。这是您可以下载
蓝灯Window 7 的链接。
https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-win7.exe

**Reply by ycjdixon** [reply]
非常感谢你通知我，WIN7终于可以继续使用蓝灯了，这几天一直可以连上。专业版用户之前已经停用好几周了。请问之后这个7.8.2WIN7版不会再自动更新到不支持WIN7的版本了吧？

**Reply by MaxMKMax** [reply]
@ycjdixon 您好。不客气! 对于造成的不便，我们深表歉意。不用担心，下一个版本发布时我们将照顾 Windows 7，并且我们将尽最大努力确保 Windows 7 正常运行。感谢您对蓝灯的支持。
